docs/man_pages/project/testing/publish-ios.md

```diff
@@ -0,0 +1,50 @@
+publish ios
+==========
+
+<% if((isConsole && isMacOS) || isHtml) { %>
+Usage | Synopsis
+---|---
+General | `$ tns publish ios [<Application> [<Username> [<Password> [<Mobile Provisioning Profile Identifier> [<Code Sign Identity>]]]]]`
+
+Builds the project and uploads it to iTunesConnect. `Application Name`, `Username` and `Password` parameters are required but if not provided, the CLI will ask for them.
```

Comment:
What is Application Name? Application?

"are required but if missing, the CLI will prompt you to provide them."